code-challenges.js

```diff
@@ -4,46 +4,345 @@
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+// const joiningTwoArrays = () => {
+//     return (myNumbers1.join(myNumbers2))
+// }
+// console.log(joiningTwoArrays())
+// //[ 3, 7, 0, 36, -9 ]
+// [ 8, -7, 42, 9, 13 ]
+// 38,-7,42,9,1378,-7,42,9,1308,-7,42,9,13368,-7,42,9,13-9
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+// const joiningTwoArrays = () => {
+//     return (myNumbers1.concat(myNumbers2))
+// }
+// console.log(joiningTwoArrays())
+// const lengthTwoArrays = () => {
+//     return (joiningTwoArrays.length())
+// }
+//no errors and I think its working need o pull a console.log
+//realized I was using a join not concat after looking at my javascript notes
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+// const joiningTwoArrays = () => {
+//     return (myNumbers1.concat(myNumbers2))
+// }
+// console.log(joiningTwoArrays())
+// const lengthTwoArrays = () => {
+//     return (joiningTwoArrays.length())
+// }
+// console.log(lengthTwoArrays())
+//this console.log causes an error
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+// var joiningTwoArrays = myNumbers1.concat(myNumbers2)
+// console.log(joiningTwoArrays)
+// console.log(joiningTwoArrays.push("test"))
+//done a lot of changes but not sure how to move on
+// the result at teh moment is they are grouped but look funky unsure how to fix
+// [ 3, 7, 0, 36, -9 ]
+// [ 8, -7, 42, 9, 13 ]
+// [
+//   3,  7,  0, 36, -9,
+//   8, -7, 42,  9, 13
+// ]
+//
+// const myNumbers1 = [3, 7, 0, 36, -9]
+// const myNumbers2 = [8, -7, 42, 9, 13]
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+// var joiningTwoArrays = myNumbers1.concat(myNumbers2)
+// console.log(joiningTwoArrays)
+// console.log(joiningTwoArrays.length)
+//array still looks funky but .length has the result of 10 moving on
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
+
+// const currentCohort = "Bravo 2022"
+
+// const currentCReversed = currentCohort.split()
+// console.log(currentCReversed.reverse())
+//[ 'Bravo 2022' ] is the current response from this 
+// forgot to write before this
+//reverse works on arrays so I used split to make it an array but I didnt use the split("") ill try that next see what it does
+// const currentCohort = "Bravo 2022"
+// const currentCReversed = currentCohort.split("")
+// console.log(currentCReversed.reverse())
+//trying with ' instead of ""
+// const currentCohort = "Bravo 2022"
+// var currentCReversed = currentCohort.split('')
+// console.log(currentCReversed.reverse())
+// console.log(currentCReversed.join())
+// 2,2,0,2, ,o,v,a,r,B
+// const currentCohort = "Bravo 2022"
+// var currentCReversed = currentCohort.split('')
+// currentCReversed.reverse()
+// console.log(currentCReversed.join(''))
+// result:
+//   2202 ovarB
+
+// //
+// recieved 2202 ovarB but you wanted Expected output: "1202 ovarB"
```

Comment:
I believe that the "1202 ovarB" is a typo in the question. You got the correct output using lines 229 -232